netlogo-core/src/main/api/PostProcessorInputFormat.scala

```diff
@@ -0,0 +1,9 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.api
+
+object PostProcessorInputFormat {
```

Comment:
Because this object lives in the `org.nlogo.api` space, maybe prefix with something to indicate it's for the BehaviorSpace/Lab code - `LabPostPorcessorInputFormat`.  Also rename the file to match.